src/app/security-group/components/sg-rule-addition-form/sg-rule-addition-form.component.ts

```diff
@@ -9,12 +9,12 @@ import {
   Validators
 } from '@angular/forms';
 import { Subscription } from 'rxjs/Subscription';
+import 'rxjs/add/operator/startWith';
```

Comment:
Let's add this to https://github.com/bwsw/cloudstack-ui/blob/master/src/polyfills.ts

Reply:
This is a redundant import, I've deleted it